hackathon/activities/srlinux-i-custom-cli/solutions/skeleton.py

```diff
@@ -13,55 +9,56 @@
 
 
 class Plugin(CliPlugin):
-    
+
     '''
         Load() method: load new CLI command at CLI startup
         Input: cli, the root node of the CLI command hierarchy
     '''
     def load(self, cli, **_kwargs):
         print("\n##########\n\n---> DEBUG: LOADING BGP-BY CLI PLUGIN\n\n##########\n")
-       
+
     '''
         _my_schema() method: construct the schema for this cli command
-        Return: schema object 
-    '''    
+        Return: schema object
+    '''
     def _my_schema(self):
         root = FixedSchemaRoot()
         ## add your code here
         return root
-   
+
     '''
         _fetch_state() method: extract relevant data from the state datastore
         Input: state, reference to the datastores
         Input: arguments, the CLI command's context
-        Return: copy of a section of the state datastore 
-    ''' 
+        Return: copy of a section of the state datastore
+    '''
     def _fetch_state(self, state, arguments):
         pass ## replace with your code
-   
+
     '''
         _populate_schema() method: fill in schema from state datastore
+        Input: state_datastore, state datastore extract
         Input: arguments, the CLI command's context
-        Return: filled in schema 
-    ''' 
-    def _populate_schema(self, arguments):
+        Return: filled in schema
+    '''
+    def _populate_schema(self, state_datastore, arguments):
```

Comment:
@zenodhaene why did you add `state_datastore` here?

Reply:
There is an inline copy of this file in the README of this activity, but the two didn't align. I simply copied the inline version (which is confirmed working) to the attached file in the solutions folder.